src/main/scala/nl/knaw/dans/easy/ps/DDM.scala

```diff
@@ -113,12 +113,35 @@ object DDM {
 
   def createComposedCreator(dictionary: Dictionary, authorFields: Iterable[(MdKey, String)]) = {
     <dcx-dai:creatorDetails>
-      <dcx-dai:author>
-        { authorFields.map(composedEntry(dictionary)) }
-      </dcx-dai:author>
+      {
+      if (isOrganization(authorFields))
+        <dcx-dai:organization>
+          <dcx-dai:name xml:lang="zh-CN">
```

Comment:
Copy from https://drivenbydata.atlassian.net/browse/EASY-916

Note:
Dit is nog niet opgelost: <dcx-dai:name xml:lang="zh-CN"> 
zh-CN is hard gecodeerd. Ik weet 't niet van waar de waarde moet ik vullen. In de DDM.scala staat overaal hard gecodeerd xml:lang="en". De zh-CN komt uit de min.xml